### PR TITLE
docs(nx): update workspace/init with pipeline changes

### PR DESCRIPTION
**Pull request by CerdenAmido**
#### 📲 What

A description of the change.

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

**Review by Danphillipz — CHANGES_REQUESTED** [review]
1. Prerequisites: The infra requires various values to be set via the stacks CLI or the create-workspace script. 
2. May need a discussion to decide on where we document the values which drive infra related files (point 1 should link to this)

File: docs/nx/workspace/init.md [hunk 1]
```diff
@@ -39,6 +40,24 @@ Files created:
 │   ├── tsconfig.base.json
 ```
 
+If `--pipelineRunner=true` is passed, the generator will also create a `build` directory:
```
[the remark this answers is not in the document]
Comment (CerdenAmido) [reply docs/nx/workspace/init.md:43]: Whoops, thank you

**Comment by CerdenAmido** [comment]
> 1. Prerequisites: The infra requires various values to be set via the stacks CLI or the create-workspace script.

To clarify my understanding on this - normally we run this generator under the hood as part of `create-stacks-workspace`. At this point, we check if `stacks.json` has certain fields, which it always should at this point, and if it doesn't we pass `--pipelineRunner=none`. We *don't* want to document this here because it's not really a prerequisite of the generator as such. The generator itself checks whether the stacks config has a `pipeline` field, or whether the `build` folder already exists, and doesn't create the pipeline folder in either case. We *do* want to document this here as it's the generator doing the work.

Is that assessment correct?

**Comment by Danphillipz** [comment]
> 

Yes this is normally ran under the hood, but can also be ran independently to 'stackify' an existing nx workspace. 

You're correct, I don't think we do want to document all of the values which this generator requires as a prereq for infra as it's common across this and the next:infrastructure generator. However I think we should at least say, in order to generate infrastructure for your application the following dependendecies must be in place (with a link to where this is documented)

**Review by Danphillipz — APPROVED** [review]

**Pull request merged**